github-app-id.yml

```diff
@@ -0,0 +1,20 @@
+- !policy
+  id: github-apps
+  body:
+      - !group
+
+      - &hosts
+        - !host
+          id: release
+          annotations:
+              authn-jwt/github/repository: quincycheng/conjur-action
```

Comment:
I know it _technically_ doesn't matter, but updating this to be `infamousjoeg/conjur-action` would eliminate potential confusion from a customer viewing the policy on this repository.

Reply:
Sure. Updated.